kumascript/macros/CSSInfo.ejs

```diff
@@ -283,7 +283,7 @@ if (name === "preview-wiki-content") {
     if (!atRule) {
         properties = properties.concat({
             name: "animationType",
-            label: localize(localStrings, "animationType")
+            label: await template("animationType")
```

Comment:
Let's avoid introducing a new macro here. Can you add the local string to this macro and build the link in line here?

Reply:
done , you can check the latest commit , thanks